### ISSUE TITLE
Wrong signature for torch.max_pool(1d, 2d, 3d) in get_testing_overrides()

### ISSUE THREAD

**Issue by volcacius** [pytorch/pytorch#45494]
## 🐛 Bug

torch.max_pool(1d, 2d, 3d) doesn't take return_indices as an argument, as seen in torch/\_\_init\_\_.pyi (while torch.nn.functional.max_pool(1d, 2d, 3d,) and torch._max_pool(1d, 2d, 3d) do) . 
However, the signature reported in torch/overrides.py/get_testing_overrides() includes return_indices as an argument.

## To Reproduce

Compare the two files mentioned above.

## Expected behavior

The signature should match.

## Environment

Applies both to 1.6.0 (_overrides.py) and current master (overrides.py).

## Additional context

Nothing.

cc @mruberry @rgommers @VitalyFedyunin @ezyang @bhosmer @smessmer @ljk53 @bdhirsh @ailzhang

**Comment by zhangguanheng66** [comment]
Is this behavior expected?

**Comment by volcacius** [comment]
According to the current [docs](https://pytorch.org/docs/stable/notes/extending.html) "torch._overrides.get_testing_overrides function returns a dictionary mapping overridable functions in the PyTorch API to dummy lambda functions that have the same signature as the original function but unconditionally return -1. "

**Comment by ezyang** [comment]
cc @hameerabbasi

**Comment by ezyang** [comment]
I think it's a bug in the overrides.

**Comment by mruberry** [comment]
@hameerabbasi What's the best label for issues related to the overrides?

**Comment by hameerabbasi** [comment]
Hi Mike, I believe one was to be created, but I don’t know if it has been yet. But if it isn’t, `module: numpy` is what we’ve used so far.

**Comment by mruberry** [comment]
Cool. Sounds good.

**Comment by volcacius** [comment]
Another set of issues, the order of ceil_mode and return_indices in the signature of torch.nn.functional.max_pool(1d, 2d, 3d) and torch.nn.functional.max_pool(1d, 2d, 3d)_with_indices is swapped.

**Comment by hameerabbasi** [comment]
I've opened PR #45983 to address this.
